core/tests/protractor/cacheSlugs.js

```diff
@@ -0,0 +1,46 @@
+// Copyright 2016 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview End-to-end tests for cache slugs.
+ */
+
+var general = require('../protractor_utils/general.js');
+
+var ERROR_PAGE_URL_SUFFIX = '/console_errors';
+
+var checkConsoleErrorsExist = function(expectedErrors) {
+  browser.manage().logs().get('browser').then(function(browserLogs) {
+    expect(browserLogs.length).toBeGreaterThan(0);
```

Comment:
Shouldn't this check that the length matches expectedErrors.length? I don't understand why we keep doing "weaker" checks -- tests should be really explicit.

If you want to make the error messages unique, that's fine, but please still err towards testing equality of things, as exactly as possible.